geomesa-utils/pom.xml

```diff
@@ -136,4 +136,24 @@
             <artifactId>specs2_2.11</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <excludes>
+                    <exclude>org/locationtech/geomesa/geomesa.properties</exclude>
+                </excludes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>org/locationtech/geomesa/geomesa.properties</include>
+                </includes>
```

Comment:
Derp.  Can't read.  My XML eyes glossed over the difference between includes and excludes:(.